modules/dcache/src/main/java/org/dcache/services/ssh2/Ssh2Admin.java

```diff
@@ -318,6 +327,66 @@ public boolean authenticate(String userName, PublicKey key,
         }
     }
 
+    private class RemoteHostValidator {
```

Comment:
I reckoned it might make sense to encapsulate all the host validation related functionalities in this inner class, but if this goes against convention or is unnecessary we can get rid of it!